materia/forms.py

```diff
@@ -270,41 +252,38 @@ class Meta:
                   'alinea',
                   'item']
 
-    def __init__(self, *args, **kwargs):
+    def clean(self):
+        if self.errors:
```

Comment:
@eduardoedson Fiquei em dúvida agora com essa condição. Se o método ``clean()`` passar sem erros -- neste caso, se existir a norma jurídica e a legislação não tiver sido cadastrada ainda -- então retornamos ``cleaned_data`` aqui:  https://github.com/interlegis/sapl/pull/380/files#diff-a0a715576fd0d95fd5c46c7cafd69583R280 

O problema é que se já tiver tido erros -- ``self.errors`` é diferente de None -- então retornamos ``self.errors``???? Outra dúvida que fiquei é se esse if realmente é necessário pq o ``clean()`` vai ser o último método do tipo ``clean_*()`` a ser chamado (certo?). Assim sendo, se, por exemplo, um dos campos obrigatórios não tiver sido preenchido então ele nem chega ao método ``clean()``, certo? Se esse for o caso, entendo que podemos deixar até pra sairmos mais rápido do método ``clean()``, mas ainda não entendi esse lance de retornar dois tipos de dados diferentes dependendo se tem erro ou não.


https://github.com/django/django/blob/ecb59cc6579402b68ddfd4499bf30edacf5963be/django/forms/forms.py#L411